src/EventTickets/Routes/GetEventForms.php

```diff
@@ -28,7 +31,9 @@ public function registerRoute()
                 [
                     'methods' => 'GET',
                     'callback' => [$this, 'handleRequest'],
-                    'permission_callback' => '__return_true',
+                    'permission_callback' => function () {
+                        return current_user_can('read');
```

Comment:
I would think that this endpoint is either public (no permissions needed), or only readable by an administrator. Checking against the `read` capability suggests that the user must be authenticated, but any authenticated user can access the information, which doesn't sound correct.

Reply:
That's a valid point. I suggest restricting this to admins who can manage forms and posts. What level of capability do you think is appropriate for this? Should we limit it to `manage_options` or consider a lower level like `edit_posts`?

Reply:
Using `manage_options` is a common fallback for "is administrator", which I usually prefer over re-using `edit_posts` (when it isn't a `post`).

Reply:
I'm concerned that users with the `give_manager` role can't manage events using `manage_options`. I'm exploring a capability like `manage_give_settings` that could meet this need for both Administrators and Give Managers.

Reply:
Ok, if you find a better capability then let's use that.

Reply:
After checking that other menu items use the `edit_give_forms` to even being displayed, the permission checks were updated to use that capability: ef40760a21e94d8b887460706f6b97d63be4915a